index.html

```diff
@@ -73,6 +73,7 @@ <h2>Entries</h2>
     </div>
   </main>
 
+  <script src="https://kit.fontawesome.com/84551c9772.js" crossorigin="anonymous"></script>
```

Comment:
I don't believe this is needed. I believe font awesome has already been imported into this project for you